packages/wallet/src/redux/asset-holders-state/state.ts

```diff
@@ -82,3 +82,29 @@ export function recordDeposit(
     newAssetHolderChannelState
   );
 }
+
+export function recordAssetTransfer(
+  assetHoldersState: AssetHoldersState,
+  assetHolderAddress: string,
+  origin: string,
+  // @ts-ignore
+  destination: string, // TODO: Should we take this into account?
```

Comment:
It's possible in future that `destination` is removed so that it is not even part of the data emitted in the event. We might decide that it is redundant information. It was introduced mainly to make testing easier, but we could get the data from somewhere else most likely. In the case of an ERC20, for example, the Token contract itself will emit a `Transfer` event when the Tokens are transferred.